feature/src/main/java/com/teamdontbe/feature/onboarding/OnboardingAdapter.kt

```diff
@@ -0,0 +1,20 @@
+package com.teamdontbe.feature.onboarding
+
+import androidx.fragment.app.Fragment
+import androidx.viewpager2.adapter.FragmentStateAdapter
+
+class OnboardingAdapter(fragmentFragment: OnboardingFragment) :
+    FragmentStateAdapter(fragmentFragment) {
+    override fun getItemCount(): Int {
+        return 4
+    }
```

Comment:
p3 : 
```suggestion
    override fun getItemCount(): Int = 4
```
로 바꾸어도 좋을 것 같습니다!

4의 경우 상수화 시켜도 좋을 것 같아요

Reply:
P3 : 훨씬 좋네요!